src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/ad/AutoplayVuap.java

```diff
@@ -138,12 +153,24 @@ private WebElement findSpeakerIcon() {
   }
 
   private void waitFor(final Predicate<AutoplayVuap> predicate, final long timeout) {
-    new FluentWait<AutoplayVuap>(this)
+    new FluentWait<>(this)
         .withTimeout(timeout, TimeUnit.SECONDS)
         .pollingEvery(1, TimeUnit.SECONDS)
         .until(predicate);
   }
 
+  private boolean isOverlayVisible() {
+    return driver.findElement(By.cssSelector(".replay-overlay")).isDisplayed();
+  }
+
+  public boolean isResolvedStateDisplayed(double defaultVideoHeight, double resolvedVideoHeight) {
+    int percentResult = (int)Math.round(100-(100/(defaultVideoHeight/resolvedVideoHeight)));
```

Comment:
Please, rename to more precise (the result of what?). Maybe even extract the whole line to separate function.